### ISSUE TITLE
Slide number does not increase for slides with chunk_reveal

### ISSUE THREAD

**Issue by loreabad6** [EvaMaeRey/flipbookr#23]
Although a `chunk_reveal` used in a `knitr` context starts on a new slide, the slide number does not increase. 
I guess it is fine when you have a slide with text right before your flipbook slide with code, as one could say, it is the continuation of the previous slide (?). But, for cases when there are two or more sequential slides with a `chunk_reveal`, then the number of slides is not reflecting how long the presentation really is. 

I adapted the minimal flipbook to show my point: 

![test](https://user-images.githubusercontent.com/10034237/85870807-d81a4380-b7cd-11ea-8325-7f4b6d8d09ed.gif)

I don't find immediately something on the `flipbookr` package that breaks the numbering, so it might be something related to `xaringan` or `revealjs`, BUT maybe I missed something?

**Comment by EvaMaeRey** [comment]
I agree with you about what the behavior should be.  Thanks for illustrating the problem so clearly!  I will probably start by asking some xaringan friends!  Thanks for this issue, Lore!  :-)

**Comment by EvaMaeRey** [comment]
So build only has one slide number.  Then, we want slide number to advance upon step one of build, else not advance.  Now it doesn't advance in any flipbook slides.  count needs to be true for first spauned slide, then false for remainder

**Comment by EvaMaeRey** [comment]
*spawned
